paasta_tools/utils.py

```diff
@@ -2246,6 +2247,13 @@ def get_auto_config_instance_type_aliases(self) -> Dict[str, str]:
         """
         return self.config_dict.get("auto_config_instance_type_aliases", {})
 
+    def get_api_client_timeout(self) -> int:
+        """
+        We've seen the Paasta API get hung up sometimes and the client not realizing this will sit idle forever.
+        This will be used to specify the default timeout
+        """
+        return self.config_dict.get("api_client_timeout", 120)
```

Comment:
it's worth double-checking that we don't have any timeouts that are under <120 in paasta status